provider/openstack/cinder.go

```diff
@@ -217,8 +217,13 @@ var _ storage.VolumeSource = (*cinderVolumeSource)(nil)
 func (s *cinderVolumeSource) CreateVolumes(ctx context.ProviderCallContext, args []storage.VolumeParams) ([]storage.CreateVolumesResult, error) {
 	results := make([]storage.CreateVolumesResult, len(args))
 	for i, arg := range args {
-		volume, err := s.createVolume(arg)
+		volume, err := s.createVolume(ctx, arg)
 		if err != nil {
+			_, denied := MaybeHandleCredentialError(err, ctx)
+			if denied {
+				errors.Annotate(err, "cannot create volume.")
```

Comment:
This code doesn't do anything. Take a look inside Annotate() - it returns an error and we are not using it anywhere here.